src/yarpdataplayer/include/worker.h

```diff
@@ -130,15 +130,23 @@ class MasterThread : public QObject,  public yarp::os::PeriodicThread
     int                     numThreads;
     double                  timePassed, initTime, virtualTime;
     bool                    stepfromCmd;
+    
+    typedef std::chrono::time_point<std::chrono::steady_clock> Moment;
```

Comment:
I'd prefer `using Moment = std::chrono::time_point<std::chrono::steady_clock>;`

Reply:
changed thank you